### PR TITLE
test: e2e contract test forward eth and mint from external contract

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-606

### CODE REVIEW

File: packages/e2e-contract-tests/playwright/e2e/ForwardHalfCustomAsset.test.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ test.describe('Forward Custom Asset', () => {
     fuelWallet = Wallet.fromMnemonic(FUEL_MNEMONIC, fuelProvider);
     await seedWallet(
       fuelWallet.address.toString(),
-      bn(1_000_000_000),
+      bn(100_000_000_000),
```
Comment (LuizAsFight) [inline packages/e2e-contract-tests/playwright/e2e/ForwardHalfCustomAsset.test.ts:30]: bn.parseUnits

File: packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ test.describe('Mint Assets', () => {
     fuelWallet = Wallet.fromMnemonic(FUEL_MNEMONIC, fuelProvider);
     await seedWallet(
       fuelWallet.address.toString(),
-      bn(1_000_000_000),
+      bn(100_000_000_000),
```
Comment (LuizAsFight) [inline packages/e2e-contract-tests/playwright/e2e/MintAsset.test.ts:28]: bn.parseUnits

File: packages/e2e-contract-tests/playwright/e2e/ForwardHalfAndExternalMint.test.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  process.env;
+
+test.describe('Forward Half ETH and Mint External Custom Asset', () => {
+  test.beforeEach(async ({ context, extensionId, page }) => {
+    await walletSetup(context, extensionId, page);
+    const fuelProvider = await Provider.create(FUEL_PROVIDER_URL!);
+    const fuelWallet = Wallet.fromMnemonic(FUEL_MNEMONIC, fuelProvider);
+    await seedWallet(
+      fuelWallet.address.toString(),
+      bn(100_000_000_000),
```
Comment (LuizAsFight) [inline packages/e2e-contract-tests/playwright/e2e/ForwardHalfAndExternalMint.test.ts:26]: bn.parseUnits

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/e2e-contract-tests/playwright/e2e/ForwardHalfAndExternalMint.test.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    // test return asset name is shown
+    await hasText(walletPage, 'Ethereum', 1);
+    // test return asset id is shown
+    await hasText(walletPage, shortAddress(BaseAssetId), 1);
+    // test return eth amount is correct
+    await hasText(walletPage, `${halfDepositAmount} ETH`);
+
+    // test mint asset name is shown
+    await hasText(walletPage, 'Unknown', 0, 5000, true);
```
[the remark this answers is not in the document]
Comment (matt-user) [reply packages/e2e-contract-tests/playwright/e2e/ForwardHalfAndExternalMint.test.ts:61]: https://linear.app/fuel-network/issue/FRO-628/different-asset-configurations

**Pull request merged**